plugins/utils/redirects.js

```diff
@@ -350,10 +350,83 @@ const redirects = `
+  /docs/current/developer-docs/integrations/sns/tokenomics/preparation /docs/current/developer-docs/daos/sns/tokenomics/preparation
+  /docs/current/developer-docs/integrations/sns/integrating/ /docs/current/developer-docs/daos/sns/integrating/
+  /docs/current/developer-docs/integrations/sns/integrating/ledger-integration /docs/current/developer-docs/daos/sns/integrating/ledger-integration
+  /docs/current/developer-docs/integrations/sns/integrating/index-integration /docs/current/developer-docs/daos/sns/integrating/index-integration
+  /docs/current/developer-docs/integrations/sns/integrating/frontend-integration /docs/current/developer-docs/daos/sns/integrating/frontend-integration
+  /docs/current/developer-docs/integrations/sns/testing/testing-before-launch /docs/current/developer-docs/daos/sns/testing/testing-before-launch
+  /docs/current/developer-docs/integrations/sns/testing/testing-locally /docs/current/developer-docs/daos/sns/testing/testing-locally
+  /docs/current/developer-docs/integrations/sns/testing/testing-on-mainnet /docs/current/developer-docs/daos/sns/testing/testing-on-mainnet
+  /docs/current/developer-docs/integrations/sns/launching/ /docs/current/developer-docs/daos/sns/launching/
+  /docs/current/developer-docs/integrations/sns/launching/launch-summary-1proposal  /docs/current/developer-docs/daos/sns/launching/launch-summary-1proposal
+  /docs/current/developer-docs/integrations/sns/launching/launch-steps-1proposal /docs/current/developer-docs/daos/sns/launching/launch-steps-1proposal
+  /docs/current/developer-docs/integrations/sns/managing/manage-sns-intro /docs/current/developer-docs/daos/sns/managing/manage-sns-intro
+  /docs/current/developer-docs/integrations/sns/managing/making-proposals /docs/current/developer-docs/daos/sns/managing/making-proposals
+  /docs/current/developer-docs/integrations/sns/managing/cycles-usage /docs/current/developer-docs/daos/sns/managing/cycles-usage
+  /docs/current/developer-docs/integrations/sns/managing/sns-asset-canister /docs/current/developer-docs/daos/sns/managing/sns-asset-canister
+  /docs/current/developer-docs/integrations/sns/managing/managing-nervous-system-parameters /docs/current/developer-docs/daos/sns/managing/managing-nervous-system-parameters
+  /docs/current/tokenomics/nns/nns-intro /docs/current/developer-docs/daos/nns/overview
+  /docs/current/tokenomics/token-holders/nns-app-quickstart /docs/current/developer-docs/daos/nns/nns-app-quickstart
+  /docs/current/tokenomics/nns/neurons-fund /docs/current/developer-docs/daos/nns/neurons-fund
+  /docs/current/tokenomics/nns/nns-staking-voting-rewards /docs/current/developer-docs/daos/nns/staking-voting-rewards
+  /docs/current/tokenomics/nns/proposal-requirements /docs/current/developer-docs/daos/nns/proposal-requirements
+  /docs/current/tokenomics/index /docs/current/developer-docs/daos/overview
```

Comment:
I think we also need a redirect from `/docs/current/tokenomics` to `overview`.
Ditto for all new `index` redirects.

Reply:
Good point! I added redirects for these.